modules/services/gpg-agent.nix

```diff
@@ -71,7 +64,15 @@ in
     };
   };
 
-  config = mkIf cfg.enable (mkMerge [
+  config = let
+    gpgInitStr = ''
+        export GPG_TTY=$(tty)
```

Comment:
This should be

```bash
export GPG_TTY
GPG_TTY="$(tty)"
```

like it was originally, otherwise the exit code from tty will be silently ignored on error.